tests/src/Interop/DllImportAttribute/Simple/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+cmake_minimum_required (VERSION 2.6)
+project (DllFileProbe)
+set(SOURCES DllFileProbe.cpp )
+
+# add the executable
+add_library (DllFileProbe SHARED ${SOURCES})
+target_link_libraries(DllFileProbe ${LINK_LIBRARIES_ADDITIONAL}) 
+
+#get_cmake_property(_variableNames VARIABLES)
```

Comment:
Do we need these commented out blocks?

Reply:
I will remove these commented lines..just copy & paste..

Reply:
Removed these comments